### PR TITLE
Use correct Javadoc syntax

### PR DESCRIPTION
**Pull request by ilammy**
While tinkering with publishing of AndroidThemis I have found out that Javadoc generator yells at me with a ton of errors like

    src/wrappers/themis/java/com/cossacklabs/themis/KeypairGenerator.java:49: error: @param name not found
             * @param key type (EC or RSA)

    [...]

    src/wrappers/themis/java/com/cossacklabs/themis/SecureSession.java:38: error: malformed HTML
                    NEGOTIATING, /** < key agreement is in progress. No data exchange possible yet. */
                                     ^
    36 errors
    36 warnings

    > Task :android:generateJavadoc FAILED

when I do

    ./gradlew :android:generateJavadoc

which is a part of publication process.

Well, this won't fly for proper publication so let's fix it. At least, make the Javadoc generator happy at first.

Make sure that all `@param` lines have an actual parameter name followed by its description. I make no effort to make it “good”, only making it correct. Improving quality of API documentation – ideally across all wrappers – is a project for another day.

Similarly, Javadoc does not support “inline” syntax for items like Doxygen-flavor documentation comments do. Just add full-fledged comments for enum variants.

## Checklist

- [X] Change is covered by automated tests
- [X] Public API has proper documentation

### CODE REVIEW

**Comment by ilammy** [comment]
Come to think of it... should we maybe have a test for this? 🤔

File: src/wrappers/themis/java/com/cossacklabs/themis/SecureCell.java [hunk 1]
```diff
@@ -625,7 +625,7 @@ public SecureCell(int mode) {
 	
 	/**
 	 * Creates new SecureCell with default master key in SEAL mode
-	 * @param default master key
+	 * @param key master key
```
Comment (vixentael) [inline src/wrappers/themis/java/com/cossacklabs/themis/SecureCell.java:628]: i love this kind of comments.. reminds me my young java dev days

**Review by vixentael — APPROVED** [review]
https://www.youtube.com/watch?v=Mk3qkQROb_k

**Comment by ilammy** [comment]
Regarding tests, I'll make sure that publishing pipeline is tested in the PR that adds Maven Central repo support. This will include generating Javadocs and will catch issues like this in the future.

**Pull request merged**